### ISSUE TITLE
Should GS mode persist between classifications?

### ISSUE THREAD

**Issue by wgranger** [zooniverse/Panoptes-Front-End#4994]
## Current behavior
Currently, the classifier clears out of GS mode after each classification is submitted. This was originally intended so users don't accidentally make a string of classifications under GS mode. 
https://github.com/zooniverse/Panoptes-Front-End/pull/1670#issuecomment-144447375

## Additional information
Some users might want to have GS persist through multiple classifications. Some ideas are included here https://github.com/zooniverse/Panoptes-Front-End/pull/4987#issuecomment-429901513

**Comment by beckyrother** [comment]
Good thinking to make sure there's plenty of visual feedback to make sure it's clear that the volunteer is in Gold Standard Mode. Would a gold border and the words Gold Standard Mode above the project name be sufficient?
Example: 
![image](https://user-images.githubusercontent.com/1392130/46963629-c7fb1880-d06b-11e8-86a5-c914ec97a810.png)
@yshish thoughts?

**Comment by yshish** [comment]
@beckyrother I would rather prefer having it displayed/highlighted somewhere around/on the 'Done' button as that is the place I focus my attention when finishing a classification - finishing the classification is the moment when I can best decide whether I am confident enough to make it a GS classification or to go with a regular one.

I don't actually focus on the project name area (or the top left area in general) during the classification as everything except the image is located on the right side. 

Hope that makes sense.

**Comment by wgranger** [comment]
With the fix recently made in #4987, the "done" button should now change once in gold standard mode (see image below). A bug previously prevented this from appearing. 

![screen shot 2018-10-16 at 9 14 55 am](https://user-images.githubusercontent.com/14099077/47022920-2388dd00-d124-11e8-8933-fa1457a41219.png)

**Comment by wgranger** [comment]
Did we arrive on a solution for this? I can make the change if we agree upon something. If gold standard mode persists through classifications, is the star by the "done" button enough to keep the user from accidentally making gold standard classifications?

**Comment by yshish** [comment]
I think if the star was of a different colour than the 'Done' word, it would be easier to be aware of the fact we're having the GS activated. 

Also, once the GS is activated, there should be some kind of a warning/note saying the GS is now persisting unless we deactivate it.

Once we leave the classification page, will the GS setting deactivate? (I would prefer that.)

**Comment by beckyrother** [comment]
Yes, the GS mode will need to be re-activated every time you return to the classification page. 

Two options for the button:
1, Extremely obvious gradient behind the Done button:
![classifier](https://user-images.githubusercontent.com/1392130/47314583-ac61b600-d607-11e8-82c4-42882643aa39.png)

2, Border around the Done button and a written warning:
![classifier](https://user-images.githubusercontent.com/1392130/47314609-c3a0a380-d607-11e8-83aa-0709058d57eb.png)

Both of these can be combined with the border around the page to make doubly sure it's clear that GS mode is active.

**Comment by yshish** [comment]
Cool. I'm actually only working with the dark theme but I guess it will look fine for both. :)

**Comment by beckyrother** [comment]
Dark theme:
![classifier](https://user-images.githubusercontent.com/1392130/47366252-ef249c00-d6a2-11e8-951a-cc4347e83c04.png)

@wgranger let me know what you need from me for this.

**Comment by wgranger** [comment]
Looks pretty neat. I think I should be able to swing that.

**Comment by yshish** [comment]
Oh, it's perfect! Doesn't even need any glitters! :+1:

**Comment by beckyrother** [comment]
That gold color is #F0B200FF on both light and dark, and fades into #F0B20000 (same color with 100% transparency). The star is Font Awesome. lmk if you need anything else!
